.github/workflows/ci.yml

```diff
@@ -1,19 +1,22 @@
 name: CI
+
 on:
   pull_request:
     types: [opened, synchronize, reopened]
   push:
     branches:
       - master
+
 env:
   IMAGE_TAG: ${{ github.sha }}
+
 jobs:
   test:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
         with:
-          submodules: true
+          submodules: 'true'
```

Comment:
Interesting. I would have assumed the boolean wouldn't be a string but it seems to be the case

Reply:
https://yaml.org/type/bool.html

Reply:
This brings back memories - of someone writing a yml file with"y" / "yes" / "true" in it with no consistency - might be worth talking with @giorgiosironi to have a preferred indication - I like "true" :)

Reply:
I was also surprised. The linter complained about the unquoted string. I have vague recollections about terraform also doing this to avoid accidental conversion to int.

Reply:
The Github action docs are even more confusing:
```
    # Whether to checkout submodules: `true` to checkout submodules or `recursive` to
    # recursively checkout submodules.
    #
    # When the `ssh-key` input is not provided, SSH URLs beginning with
    # `git@github.com:` are converted to HTTPS.
    #
    # Default: false
    submodules: ''
```
Sure that this is a boolean and not a string in all cases? Was this a Github action linter or a YAML linter?